pwa-devdocs/scripts/create-reference-docs/config/index.js

```diff
@@ -5,7 +5,7 @@ let files = [].concat(peregrineFiles, veniaConceptFiles);
 
 module.exports = {
     baseGitHubPath:
-        'github.com/magento-research/pwa-studio/blob/master/',
+        'github.com/magento/pwa-studio/blob/master/',
```

Comment:
```suggestion
        'github.com/magento/pwa-studio/blob/develop/',
```
Suggesting we link to `develop` since we don't update `master` enough which often leads to 404

Reply:
Fair, but this will create docs for "unreleased" code. Are you sure you want to do this?

Things we release to `master` are "official". Things on `develop` are not.

Reply:
yeah, lets do `develop` for now. it will prevent broken links due to the change from `src` to `lib`.

See the **View Source** link in: https://magento.github.io/pwa-studio/peregrine/reference/hooks/useDropdown/